src/50_Sample_AMPs/Live_Blog.html

```diff
@@ -0,0 +1,129 @@
+    -->
+    <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
+    <script async custom-element="amp-live-list" src="https://cdn.ampproject.org/v0/amp-live-list-0.1.js"></script>
+    </script>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <style amp-custom>
+  .carousel .slide > amp-img > img {
+  object-fit: contain;
+  }
+ .logo > a {
+   font-size: 16px;
+   font-weight: 500;
+   color: white;  
+   text-transform: uppercase;
+ }
+ .heading {
+   padding-bottom: 8px;
+ }
+ .heading > #summary {
+   font-weight: 500;
+ }
+ .heading > small {
+   color: #656565;
+ }
+
+ .heading .date {
+    color: #a8a3ae;
+    font-size: 12px;
+ }
+.blog {
+      padding: 0;
+      min-width: 70%;
+      background: #fff;
+}
+.blog p {
+      padding: 8px;
+      margin: 0;
+}
+
+.brand {
+  color: #6f757a;
+}
+.blog-item-info {
+  background: #f4f4f4;
+  border: solid #dcdcdc;
+  margin: 16px;
+}
+#live-list-update-button {
+  top: 60px;
+  position: fixed;
+  left: 50%;
+  transform: translateX(-50%);
+}
+  </style>
+
+  </head>
+  <body>
+
+    <!-- -->
+    <div class="logo">
+      <a href="/"> AMP by Example</a>
+    </div>
+
+    <div class="heading">
+      <h1>A sample Live Blog</h1>
+      <p><small class="date">18 July 2016</small></p>
+      <p id="summary">This is a sample article demonstrating how to write a live blog in AMP. It demonstrates the usage of amp-live-list component which allows to create live blogs.</p>
+    </div>
+
+    <amp-live-list
+    layout="container"
+    data-poll-interval="15000"
+    data-max-items-per-page="5"
+    id="amp-live-list-insert-blog">
+    <button id ="live-list-update-button" update on="tap:amp-live-list-insert-blog.update" class="button button-primary button-update">You have updates</button>
+    <div items>
+      {{range .BlogItems}}
+          <div id="{{.Text}}" data-sort-time="{{.Timestamp}}">
+            <h2>{{.Text}}</h2>
+            <div class="blog-item-info">
+              <p itemscope="" itemtype="http://schema.org/Person" itemprop="author">
```

Comment:
why do we need metadata here? "itemscope="" itemtype="http://schema.org/Person" itemprop="author""